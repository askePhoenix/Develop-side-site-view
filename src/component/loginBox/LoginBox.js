import React from 'react'

function LoginBox() {
    return (
        <div>
            <label>ID : </label>
            <input type="text"></input>
            <label>Password : </label>
            <input type="text"> </input>
        </div>
    );
}

export default LoginBox;
