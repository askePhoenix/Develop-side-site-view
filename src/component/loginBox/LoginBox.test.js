import React from 'react';
import renderer from 'react-test-renderer';
import LoginBox from "./LoginBox";


test('LoginBox test', () => {
    const component = renderer.create(
        <LoginBox />,
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
});
