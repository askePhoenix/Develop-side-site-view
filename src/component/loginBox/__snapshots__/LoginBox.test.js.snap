// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoginBox test 1`] = `
<div>
  <label>
    ID : 
  </label>
  <input
    type="text"
  />
  <label>
    Password : 
  </label>
  <input
    type="text"
  >
     
  </input>
</div>
`;
